pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -89,7 +73,6 @@ def create_pep_zero(app: Sphinx, env: BuildEnvironment, docnames: list[str]) ->
     env.found_docs.add(pep_zero_filename)
 
     # Create peps.json
-    pep0_json = create_pep_json(peps)
-    out_dir = Path(app.outdir) / "api"
-    out_dir.mkdir(exist_ok=True)
-    Path(out_dir, "peps.json").write_text(pep0_json, encoding="utf-8")
+    json_path = Path(app.outdir, "api", "peps.json").resolve()
+    json_path.parent.mkdir(exist_ok=True)
+    json_path.write_text(create_pep_json(peps), encoding="utf-8")
```

Comment:
I'm not sure I really understand the pressing need to rewrite all this when there were no functional changes in or near this line, and the previous form was perfectly valid and does exactly the same thing...it just seems like churn to me, but maybe I'm missing something important here.

Reply:
A (small) speed up as we only sort the large list of PEPs once.

A

Reply:
I don't follow, sorry. Maybe I'm missing something, but I don't see how any of the changes here have anything to do with that, as no sorting is performed within this block and `create_pep_json` is called once both times in the same way 

(Pedantic note: Other than `create_pep_json()` is not bound to a name first before using it, which saves a few hundred kB of memory for the few ≈milliseconds it is alive while the path is checked, and perhaps on the order of microseconds on the fast local name lookup.)